exports/Taskfile.dist.yaml

```diff
@@ -53,12 +53,12 @@ tasks:
 
       - |
         echo -e "\n"
-        if [[ -z "{{.TASKIT_LOCAL_PATH}}" ]]; then
```

Comment:
Ah, I was sure I saw `-n`, not `-z` somewhere. I tested that! 😆

Reply:
It was somewhere... Not sure what happened, but I messed it up!

Reply:
I blame bash for their unreadable, horrible conditionals.